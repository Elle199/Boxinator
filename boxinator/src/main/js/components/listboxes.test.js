import React from 'react';
import { shallow } from 'enzyme';

import BoxList from './listboxes';

describe('BoxList', () => {
    it('should render correctly in "debug" mode', () => {
        const component = shallow(<BoxList debug />);

        expect(component).toMatchSnapshot();
    });
});