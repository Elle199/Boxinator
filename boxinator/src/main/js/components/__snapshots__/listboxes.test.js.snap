// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Box should create a box single row 1`] = `
<tr>
  <td>
    Max
  </td>
  <td>
    50 kg
  </td>
  <td>
    <input
      disabled={true}
      type="color"
      value="#ffff64"
    />
  </td>
  <td>
    Sweden
  </td>
  <td>
    65 SEK
  </td>
</tr>
`;

exports[`BoxList should create empty table 1`] = `
<div
  className="displayTable"
>
  <table>
    <thead>
      <tr>
        <th>
          Receiver
        </th>
        <th>
          Weight
        </th>
        <th>
          Color
        </th>
        <th>
          Destination
        </th>
        <th>
          Shipping cost
        </th>
      </tr>
    </thead>
    <tbody />
  </table>
</div>
`;
